compiler/luci/pass/src/ForwardTransposeOpPass.cpp

```diff
@@ -179,8 +177,7 @@ bool has_single_element(const luci::CircleConst *node)
 class EBOWithConstPattern final : public CircleNodeMutableVisitor<bool>
 {
 private:
-  // TODO Rename this to has_commutative_pattern
-  template <typename CIRCLE_OP_PTR> bool has_pattern(CIRCLE_OP_PTR node)
+  template <typename CIRCLE_OP_PTR> bool has_commutative_xy(CIRCLE_OP_PTR node)
```

Comment:
This function is only valid for commutative Ops, and input names have to be `x` and `y`.